src/jrd/replication/Config.cpp

```diff
@@ -83,7 +83,7 @@ namespace
 	{
 		string msg;
 		msg.printf("%s specifies %s: %s", key.c_str(), type.c_str(), value.c_str());
-		raiseError(msg.c_str());
+		raiseError("%s", msg.c_str());
```

Comment:
May be just eliminate `msg` completely moving whole formatting into `raiseError`?..

Reply:
Done